### DISCUSSION TITLE
Feature Request: Disable for Bookmarks

### DISCUSSION THREAD

**Discussion by utmorpheus** [uBlockOrigin/uBlock-issues#2395] [Ideas] [answered]
After wiping my work laptop due to Microsoft365/Intune Authentication being the hot mess it is, I remembered how much of a pain it is to set personalization back up again. This includes the dozens of business-based bookmarks that I use in multiple browsers. Many of these sites are script heavy and uBlock will stop certain features from working unless disabled (like in the Exchange admin centre). This got me thinking that a toggle in the settings to disable uBlock on bookmarks would be an amazing feature for people like me who use a certain browser for business needs, and have a separate one for personal browsing. 95% of the bookmarks I have in Edge and Chrome are ad-free but have functionality that uBlock flags as bad/ad-like behavior.

**Comment by gorhill** [comment, marked as answer]
Declined, I don't think that's a good idea as I see this as feature creep too remote from uBO's primary purpose.
